app/assets/stylesheets/home.scss

```diff
@@ -16,4 +16,21 @@
   padding: 2px;
   color: red;
   display: table;
-}
\ No newline at end of file
+}
+
+div.random_card_for_check {
+  margin-top: 50px;
+  padding-bottom: 20px;
+  background-color: #f0ad4e;
+  border: 3px solid #398439;
+  border-radius: 10px;
```

Comment:
Rule declaration should be preceded by an empty line<br>Avoid qualifying class selectors with an element.